### PR TITLE
Mdv2 combobox

### PR DESCRIPTION
**Pull request by NotNestor**
## Description
Update combobox
Override global color tokens for MDV2

## Related Issue

[CX-15169](https://jira-eng-sjc12.cisco.com/jira/browse/CX-15169)

## Screenshots:

Screenshot 2024-06-26 at 10 24 22
Screenshot 2024-06-26 at 10 24 30

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: web-components/src/wc_scss/themes/global--mdv2.scss [hunk 1]
```diff
@@ -1,4 +1,86 @@
...
+  --md-primary-one-bg-color: #{$mds-color-theme-background-solid-secondary-normal};
+  --md-primary-two-bg-color: #{$mds-color-theme-background-solid-tertiary-normal};
+
+  --md-secondary-bg-color: #{$mds-color-theme-background-solid-secondary-normal};
+  --md-secondary-one-bg-color: #{$mds-color-theme-background-secondary-normal};
+  --md-secondary-two-bg-color: #{$mds-color-theme-background-secondary-hover};
+  --md-secondary-three-bg-color: #{$mds-color-theme-background-secondary-active};
+
+  --md-tertiary-bg-color: #{$mds-color-theme-background-solid-tertiary-normal};
+  // --md-tertiary-one-bg-color: #{$md-gray-95};
```
Comment (ctummon) [inline web-components/src/wc_scss/themes/global--mdv2.scss:24]: Unused comments, more below

File: web-components/src/wc_scss/themes/global--mdv2.scss [hunk 2]
```diff
@@ -1,4 +1,86 @@
...
+
+  --md-nav-tab-hover-bg-color: #{$mds-color-theme-button-secondary-hover};
+  --md-nav-tab-active-bg-color: #{$mds-color-theme-button-secondary-active-normal};
+
+  --md-menu-overlay-border-color: #{$mds-color-theme-outline-secondary-normal};
+  
+  --md-dynamic-layout-button-border-color: #{$mds-color-theme-outline-button-normal};
+  --md-dynamic-layout-button-inverted-border-color: #{$mds-color-theme-inverted-outline-button-normal};
+  
+  // --md-presence-active-bg-color: #{$md-green-50};
```
[the remark this answers is not in the document]
Comment (NotNestor) [reply web-components/src/wc_scss/themes/global--mdv2.scss:69]: I left them as a reminder to follow up in another PR

File: web-components/src/wc_scss/themes/global--mdv2.scss (hunk 1 above)
Comment (NotNestor) [reply web-components/src/wc_scss/themes/global--mdv2.scss:24]: Left to follow up and update in another PR. Need to find out what the appropriate mapping is but i think having so many updated is worth committing

**Review by ctummon — APPROVED** [review]
Looks good to me

**Pull request merged**